app/components/provider_interface/application_choice_header_component.html.erb

```diff
@@ -36,10 +38,12 @@
             Make a decision
           </h2>
           <p class="govuk-body">
-            <% if time_is_today_or_tomorrow?(application_choice.reject_by_default_at) -%>
-              You have until <%= time_today_or_tomorrow(application_choice.reject_by_default_at) %> to respond to this application. Otherwise it will be automatically rejected.
+            <% if application_choice.reject_by_default_at.today? -%>
+              This application will be automatically rejected at the end of today (<%= time_today_or_tomorrow(application_choice.reject_by_default_at) %>) if you do not make a decision.
+            <% elsif time_is_tomorrow?(application_choice.reject_by_default_at) -%>
+              This application will be automatically rejected at the end of tomorrow (<%= time_today_or_tomorrow(application_choice.reject_by_default_at) %>) if you do not make a decision.
```

Comment:
Same as above
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4484/files#r612361779